### ISSUE TITLE
libretro: error=14 when reading from uncompressed CHD

### ISSUE THREAD

**Issue by negativeExponent** [rtissera/libchdr#29]
cmd line used : chdman cratecd -c none -i <src.cue> -o <dest.chd>

As of current libretro libchdr, it does not support uncompressed chd files. updating it will fix this bu not sure if this is known already, but reading audio tracks off from an uncompressed CHD will cause **[libretro ERROR] chd_read_sector failed lba=16410 error=14** to be displayed despite game running.

![libchdr](https://user-images.githubusercontent.com/54053706/93882469-d6034a80-fd12-11ea-854e-002f51ebf48e.png)

Original post:
https://github.com/libretro/beetle-saturn-libretro/issues/163#issuecomment-696682686

**Comment by rtissera** [comment]
Agreed, we need to have libchdr updated from upstream to libretro. 
Please work together for that.
Even better if it could be copy/pasted in every core...

**Comment by negativeExponent** [comment]
i can work this by hand/manually but to be honest a better way of syncing upstream to libretro is more preferable. there is another post about twinaphex discussion about suggested changes? in anycase i think you guys are the only once who can discuss the matter any further, i just sync and play :+1: 

speaking of sync there might be some needed C89 changes needed for the upsteam:
e.g.: https://github.com/libretro/beetle-pce-fast-libretro/pull/163/commits/ecb269bf05219bb48f92a0c539d4ed53ddfe8a6a

**Comment by rtissera** [comment]
Is it still relevant or can I close the issue ?

**Comment by Sanaki** [comment]
I can't reproduce this issue. Checked logs on Beetle PCE FAST (Motteke Tamago), Beetle Saturn (Sonic R), and Beetle PSX HW (Wild 9) using uncompressed chds generated with chdman 0.236. It appears to have been fixed at some point.

**Comment by rtissera** [comment]
ok, let's close this
